content/sifixes/src/header/si/si_gflags.uc

```diff
@@ -134,12 +134,18 @@ enum Monitor_traitor_subplot
 	TELL_SPEKTOR_ABOUT_TRAITOR = 0xB3,
 	KNOW_MARSTEN_IS_TRAITOR = 0x93,
 	KNOW_SPEKTOR_IS_TRAITOR = 0x94,
+	CAN_ACCUSE_MARSTEN = 0x36,
 	PROVED_MARSTEN_IS_TRAITOR = 0x38,
 	PROVED_SPEKTOR_IS_TRAITOR = 0x92,
 	SPEKTOR_ADMITS_HIS_CRIMES = 0xB4,
 	HAVE_SPEKTOR_KEY = 0xC1,
 	FOUND_CANTRAS_FATHER = 0xC6,
-	TOLD_HARNNA_FOUND_SCROLL = 0xC7
+	TOLD_HARNNA_FOUND_SCROLL = 0xC7,
+	POMDIRGUN_IS_DEAD = 0xCC
+	// If set, changes or adds conversation with Harnna, Shazzana, Standarr, Templar, Brendann, Caladin,
```

Comment:
I think these comment lines would be better off above the enum entry they explain. As a general rule, the explanation should be before the thing explained unless it is short enough that a comment at the end is enough